src/FashionMNIST/reader.jl

```diff
@@ -0,0 +1,13 @@
+import ..MNIST.Reader
+
+Reader.set_msg_prompt("""
+    Dataset: THE FashionMNIST DATABASE of fashion products
+    Authors: Han Xiao, Kashif Rasul, Roland Vollgraf
+    Website: https://github.com/zalandoresearch/fashion-mnist
+
+    Paper: Han Xiao, Kashif Rasul, Roland Vollgraf "Fashion-MNIST: a Novel Image Dataset for Benchmarking Machine Learning Algorithms."
+
+    The files are available for download at the offical website linked above.
+    """)
+
+Reader.set_baseurl("http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/")
```

Comment:
I like the idea, but how predictable is it to overwrite a global variable? Can one load both datasets in the same session?

Reply:
Good point, hmhmh. Let me think a bit more, maybe I can come up with something better, otherwise I'll revert to the first commit